$NetBSD: patch-libcanlock-3.pc.in,v 1.1 2024/04/29 12:20:34 micha Exp $

Add main include directory to Cflags.
The documentation specifies to include headers with "lincanlock-3/"
prefix.

--- libcanlock-3.pc.in.orig	2021-08-15 11:15:39.000000000 +0000
+++ libcanlock-3.pc.in
@@ -8,5 +8,5 @@ Name: @PACKAGE@
 Description: Library for creating and verifying RFC 8315 Netnews Cancel-Locks
 URL: https://micha.freeshell.org/libcanlock/
 Version: @VERSION@
-Cflags: -I${includedir}/libcanlock-3
+Cflags: -I${includedir} -I${includedir}/libcanlock-3
 Libs: -L${libdir} -lcanlock
